tests/utils.go

```diff
@@ -0,0 +1,17 @@
+package tests
```

Comment:
Shall we utilise already existing `testutils` directory rather than creating a new file here?

Reply:
Sure.